solar-system/src/org/gearvrf/solarsystem/SolarViewManager.java

```diff
@@ -181,17 +190,32 @@ private void counterClockwise(GVRSceneObject object, float duration) {
                 0.0f, 0.0f, 0.0f));
     }
 
-    private void clockwise(GVRSceneObject object, float duration) {
-        setup(new GVRRotationByAxisWithPivotAnimation( //
-                object, duration, -360.0f, //
-                0.0f, 1.0f, 0.0f, //
-                0.0f, 0.0f, 0.0f));
-    }
-
     private void clockwise(GVRTransform transform, float duration) {
         setup(new GVRRotationByAxisWithPivotAnimation( //
                 transform, duration, -360.0f, //
                 0.0f, 1.0f, 0.0f, //
                 0.0f, 0.0f, 0.0f));
     }
+
+    public void applySceneLightMap(GVRScene scene, GVRContext gvrContext) {
```

Comment:
There is now a way to integrate lightmaps with dynamic lighting. Multiple light support added in GearVRf 3.0 dynamically creates a shader based on your material and lighting properties. If you have a lightmap texture in your material, it will apply the light map and ignore the light sources.

For applying lightmaps to the objects in the scene, I suggest we make a helper class for doing stuff like that. Maybe it could be a general scene optimization utility. One of the optimizations would be to apply lightmaps instead of using dynamic lighting. It could also do stuff like spatially sort the scene for more optimal picking, combine static geometries, etc.